common/common.go

```diff
@@ -486,3 +487,30 @@ func GetExitStatus(err error) (int, error) {
 	}
 	return -1, err
 }
+
+// reference to GOARM, needs to be globally, if in GetArch() function it resolves to zero
+//go:linkname goarm runtime.goarm
```

Comment:
According to the doc, you should also import "unsafe" for this annotation to be effective.